pkg/patterns/declarative/reconciler.go

```diff
@@ -134,6 +136,40 @@ func (r *Reconciler) reconcileExists(ctx context.Context, name types.NamespacedN
 	if err != nil {
 		return reconcile.Result{}, err
 	}
+
+	// dynamic config
+	dynamicClientset, err := dynamic.NewForConfig(r.config)
+	if err != nil {
+		log.Error(err,"Unable to create dynamic client")
+		return reconcile.Result{}, err
+	}
+
+	newItems := []*manifest.Object{}
+	for _, obj := range objects.Items {
+
+		// Uses unsafe method?? Is it safe?
+		getOptions := metav1.GetOptions{}
+		gvk, _ := meta.UnsafeGuessKindToResource(obj.GroupVersionKind())
+		ns := obj.UnstructuredObject().GetNamespace()
+		unstruct, err := dynamicClientset.Resource(gvk).Namespace(ns).Get(context.Background(),
+			obj.Name, getOptions)
+		if err != nil{
+			log.WithValues("name", obj.Name).Error(err, "Unable to get resource")
```

Comment:
We probably also special-case `apierrors.IsNotFound(err)`; that's going to be the error for every object for a new CR and it's not work logging for IMO.